pxr/imaging/garch/CMakeLists.txt

```diff
@@ -1,17 +1,24 @@
 set(PXR_PREFIX pxr/imaging)
 set(PXR_PACKAGE garch)
 
-if (NOT ${PXR_ENABLE_GL_SUPPORT})
+if ((NOT ${PXR_ENABLE_GL_SUPPORT}) AND (NOT APPLEIOS))
```

Comment:
is the message below more correct? i.e. should this be and (NOT PXR_ENABLE_METAL_SUPPORT)?